### ISSUE TITLE
Graph - Population pyramid

### ISSUE THREAD

**Issue by AsaArrhen** [PxTools/pxtools-project#12]
As a user of PxWeb I wish to see my data as a population pyramide when selecting that option in the Display menu.

How it should look in the interface (a more detailed description of WCAG etc. can be found in https://github.com/PxTools/pxtools-project/issues/6):

<img width="736" height="903" alt="Image" src="https://github.com/user-attachments/assets/1b25e0c3-7d5b-42b5-b885-af4932433fd1" />

<br>

Components and icons:
- [Component to be used (ActionItem, large, toggleable), Storybook](https://pxweb2.pages.dev/storybook/?path=/story/components-actionitem--large-toggleable)
- [Icons to be used, Storybook](https://pxweb2.pages.dev/storybook/?path=/docs/components-actionitemicon--docs)

<br>

**Rules for the population pyramid** 
For one variable, two values must be chosen (forms the x-axis). Another variable can have an unlimited number of values (forms the y-axis). For other variables maximum one value can be selected.

<br>

Language:
- Norsk: Befolkningspyramide
- Svensk: Befolkningspyramid
- Engelsk: Population pyramid

<br>

**Acceptance criteria**

- Selecting the population pyramide option shows a population pyramide, if the rules of the population pyramid are fulfilled
- If the rules are not fulfilled, a message shows instead (@Maren-SSB is working on it)
- The chart shows with the chosen data
- When pivoting, the graph changes

**Comment by Maren-SSB** [comment]
I need to write text for the alert and also assess whether the current visualisation (table or another chart type) should be kept or replaced with an empty state.
